docs/cellxgene_census_schema.md

```diff
@@ -833,12 +851,42 @@ Cell metadata MUST be encoded as a `SOMADataFrame` with the following columns:
   <tr>
     <td>tissue</td>
   </tr>
+  <tr>
+    <td>nnz</td>
+    <td>int64</td>
+    <td>For this observation, the number of non-zero measurements in the `X['raw']` counts (raw) matrix.</td>
+  </tr>
+  <tr>
+    <td>n_measured_vars</td>
+    <td>int64</td>
+    <td>For this observation, the number of features present in the source H5AD (sum of feature presence matrix).</td>
+  </tr>
+  <tr>
+    <td>raw_sum</td>
+    <td>float32</td>
+    <td>For this observation, the sum of the count (raw) matrix values.</td>
+  </tr>
+  <tr>
+    <td>raw_mean</td>
+    <td>float32</td>
+    <td>For this observation, the mean of the count (raw) matrix values.</td>
+  </tr>
+  <tr>
+    <td>raw_variance</td>
+    <td>float32</td>
+    <td>For this observation, the variance of the counts (raw) matrix values.</td>
+  </tr>
 </tbody>
 </table>
 
 
 ## Changelog
 
+### Version 1.1.0
```

Comment:
Depending on which lands first this or #618 We'll need to adjust accordingly -- one will need to be 1.2.0